modules/apps/object/object-test/src/testFunctional/paths/CreateObject.path

```diff
@@ -155,6 +155,11 @@
 		<td>//input[contains(@name,'r_relationship_c_customObjectId') and contains(@value,'${key_entry}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>SELECT_DELETION_TYPE</td>
+		<td>//select[@id='_com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_deletionType']</td>
+		<td></td>
+	</tr>
```

Comment:
Hi @vsigor 

You don't need to add this path for this PR, this path already exists. You can check him on the `CreateObject.path` line: 169